stumpy/mmotifs.py

```diff
@@ -0,0 +1,187 @@
+import numpy as np
+import logging
+from . import core, config, mdl, match
+
+logger = logging.getLogger(__name__)
+
+
+def mmotifs(
+    T: np.ndarray,

```

Comment:
Can you please re-arrange this list of inputs to be in the same order as `motifs()`?

Reply:
```
def motifs(
    T,
    P,
    min_neighbors=1,
    max_distance=None,
    cutoff=None,
    max_matches=10,
    max_motifs=1,
    atol=1e-8,
    normalize=True,
    p=2.0,
):
```

Reply:
I skipped the parameters `normalize` and `p` so far since I have never worked with function decorators before. I'll try to watch some tutorials on them.

Reply:
I think I grasped the main concept now. Seems like we also need an `aamp_mmotifs` function, am I right?

Reply:
Alright! I think it would be best if  I do this after the `mmotifs` function is working sufficiently well, since I otherwise would have to change some parts of it twice.

Reply:
Agreed!